test/unit/spec/common/cellComponents/filePathWidget-Spec.js

```diff
@@ -77,69 +70,57 @@ define([
             });
         });
 
-        it('should start and render itself', function () {
-            return this.filePathWidgetInstance
-                .start({
+        xdescribe('the started widget', () => {
```

Comment:
Commenting out these tests -- they pass sometimes but not others. 😖

Reply:
😖